pages/items/[id]/index.js

```diff
@@ -0,0 +1,45 @@
+import useSWR from "swr";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+import ItemDetails from "@/components/ItemDetails";
+
+export default function DetailsPage() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { id } = router.query;
+
+  const { data: item, mutate, isLoading, error } = useSWR(`/api/items/${id}`);
+
+  async function handleStatus() {
+    const newItem = { ...item, inDiscuss: !item.inDiscuss };
+    await fetch(`/api/items/${id}`, {
+      method: "PUT",
+      body: JSON.stringify(newItem),
+    });
+    mutate(newItem);
+  }
+
+  useEffect(() => {
+    if (!isReady) return;
+  }, [isReady]);
```

Comment:
this useEffect has no effect ;)

Reply:
> this useEffect has no effect ;)

Can you tell me please how I have to handle isReady? I have no idea more...

Reply:
> this useEffect has no effect ;)

It has effect. Without handle isReady this way I get errors.